starboard/android/shared/file_internal.cc

```diff
@@ -112,10 +112,17 @@ bool IsAndroidAssetPath(const char* path) {
 
 AAsset* OpenAndroidAsset(const char* path) {
   if (!IsAndroidAssetPath(path) || g_asset_manager == NULL) {
+    SB_DLOG(WARNING) << "Unable to open from Android Asset Manager: " << path;
```

Comment:
Change those to `SB_LOG` please so that they are always present, outside of debug builds as well.

Reply:
Sure. it is updated to use `SB_LOG`